### ISSUE TITLE
_  in app names

### ISSUE THREAD

**Issue by plasorak** [DUNE-DAQ/flxlibs#73]
https://github.com/DUNE-DAQ/flxlibs/blob/2b4adc6f4bc8853033ae96a50f9f03d456ee5818/scripts/felixcardcontrollerconf_gen.py#L88 An application with a "`_`" in its name won't boot with k8s. @roland-sipos is there a good reason this line was changed from `nickname = nickname.replace('-', '')` to the current one?

**Comment by wesketchum** [comment]
@plasorak fixed now in dunedaq-v3.2.0? 

https://github.com/DUNE-DAQ/flxlibs/blob/371a4a2b9e61ac5fb45cfdc885536be1db356850/scripts/felixcardcontrollerconf_gen.py#L79

Well, but maybe not since 
```
(dbt-pyvenv) [wketchum@np04-srv-024 ~]$ which felixcardcontrollerconf_gen.py 
/cvmfs/dunedaq.opensciencegrid.org/spack/releases/dunedaq-v3.2.0/spack-0.18.1-gcc-12.1.0-b1/spack-0.18.1/opt/spack/gcc-12.1.0/flxlibs-v1.7.5-v26am236uuptqyt4lgc27lyk2lzn5wmt/bin/felixcardcontrollerconf_gen.py
```

@roland-sipos can you comment?

**Comment by plasorak** [comment]
Yes this is fixed, and daqconf won't let you do this anymore: hhttps://github.com/DUNE-DAQ/daqconf/blob/v5.5.3/python/daqconf/core/conf_utils.py#L397
